tests/plugins/widget/manual/contextmenushortcuts.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.11.4, bug, 1901, widget, accessibility
+@bender-ui: collapsed
+@bender-ckeditor-plugins: widget, wysiwygarea, contextmenu
+
+## Keyboard shortcuts for widgets' context menu
+
+### Scenario 1: `SHIFT + F10`
+
+1. Select widget in editor by left-click.
```

Comment:
It should be enough to rewrite it into `Focus the widget.` for small simplification.